packages/platform/api/prisma/schema.prisma

```diff
@@ -23,6 +23,10 @@ model User {
   apiKeys       ApiKey[]
   createdAt     DateTime  @default(now())
   updatedAt     DateTime  @updatedAt
+  githubId          String?   @unique
+  githubUsername    String?
+  githubAccessToken String?
```

Comment:
why are we storing access token? should this not be handled at FA level?

do we need `githubId`? can we just not create a simple `userId` and link to a gh username?